### DISCUSSION TITLE
MSCLoader.ModLoader.GetMod

### DISCUSSION THREAD

**Discussion by Pete9xi** [piotrulos/MSCModLoader#218] [Help me] [answered]
With this now being Obsolete is there a replacement method?
this is how i was using it as part of my mod

``` public override void CustomSaveSaving(Mod mod, string saveFileName)
        {
            var ModID = MSCLoader.ModLoader.GetMod("modID");
            SaveLoad.WriteValue(ModID, partId+"isBlownVal", isBlown);
            SaveLoad.WriteValue(ModID, partId+ "lampHoursVal", lampHours);

        }`

**Comment by piotrulos** [comment, marked as answer]
but why like this? just instead `SaveLoad.WriteValue(ModID, partId+"isBlownVal", isBlown); `do  `SaveLoad.WriteValue(this, partId+"isBlownVal", isBlown);` You already in same instance when saving so no need to get it.
